assets/src/scripts/components/dailyValueHydrograph/index.spec.js

```diff
@@ -116,7 +116,7 @@ describe('components/dailyValueHydrograph/index', () => {
         });
 
         window.requestAnimationFrame(() => {
-            expect(testDiv.select('.graph-container').selectAll('.hydrograph-container').size()).toBe(2);
+            expect(testDiv.select('.graph-container').selectAll('.hydrograph-container').size()).toBe(3);
             done();
         });
     });
```

Comment:
Could you also add a test to make sure the graph-brush is rendered (similar to what you added for the legend)?